manifest/com.zahndy/ColliderGrabbableDeleter/info.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "ColliderGrabbableDeleter",
+	"id": "com.zahndy.ColliderGrabbableDeleter",
+	"description": "ObjectRoot button that removes MeshColliders And Grabbables of Child slots",
+	"tags": [
+        "MeshColliders",
+        "Grabbables",
+        "Grabbable"
+    ],
+	"category": "Technical Tweaks",
+	"sourceLocation": "https://github.com/zahndy/ColliderGrabbableDeleter",
+	"versions": {
+		"1.0.1": {
+			"releaseUrl": "https://github.com/zahndy/ColliderGrabbableDeleter/releases/tag/1.0.1",
+			"artifacts": [{
+				"url": "https://github.com/zahndy/ColliderGrabbableDeleter/releases/download/1.0.1/ColliderGrabbableDeleter.dll",
```

Comment:
Probably worth a look at using `GetComponentsInChildren<>()` on the target slot instead of iterating and comparing names yourself, it is part of Slot and will return a list of all of the target component. ie `var meshColliders = slot.GetComponentsInChildren<MeshCollider>();` will just be all of that component.